### PR TITLE
docs(tokens): DLT-1635 add tokens examples for each category

### PR DESCRIPTION
**Pull request by ninamarina**
# Add tokens examples for each category

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

- [x] Documentation

## :book: Jira Ticket
[DLT-1635]

## :book: Description
Adds tokens examples for each category, according to these designs: https://www.figma.com/design/Kun0him7tdf4i7oR9wjnH9/Dialtone-Web?node-id=2424-17800&m=dev

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

For all CSS changes:

- [x] I have used design tokens whenever possible.
- [x] I have considered how this change will behave on different screen sizes.
- [x] I have visually validated my change in light and dark mode.
- [x] I have used gap or flexbox properties for layout instead of margin whenever possible.

## :camera: Screenshots / GIFs
Some examples:

image

image

image

image

[DLT-1635]: https://dialpad.atlassian.net/browse/DLT-1635?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by francisrupert** [comment]
🥇 Fricking awesome! I know you're in Draft mode still, though the theming toggle will have to be accounted for. 

https://github.com/dialpad/dialtone/assets/1165933/b236a4ac-5842-411e-bdf0-34213febc714

**Comment by braddialpad** [comment]
Probably out of the scope of this PR and more of a question for @francisrupert, but I noticed these 2 tokens are very low contrast in dark mode. Is this expected or is there something wrong with the tokens?

Screenshot 2024-05-23 at 9 37 12 AM

File: apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokenTree.vue
Comment (braddialpad) [inline apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokenTree.vue:1]: Are we able to set a fixed width for the first column so we don't get alignment issues like the below?

Screenshot 2024-05-23 at 9 43 07 AM

**Review by braddialpad — COMMENTED** [review]
Wow this makes our tokens page looks really complete! great work. No issues with the code itself really.

File: apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokenTree.vue
Comment (ninamarina) [reply apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokenTree.vue:1]: good catch, fixed in last commit

**Review by braddialpad — COMMENTED** [review]
Should we just show the same examples for negative size / space that we do for positive size / space?

File: apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokenTree.vue
Comment (braddialpad) [reply apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokenTree.vue:1]: looks great, thanks!

**Comment by ninamarina** [comment]
> Should we just show the same examples for negative size / space that we do for positive size / space?

Good idea, I think that makes sense. @fede-dp added you as reviewer in case you have some input on this, since I don't see the negatives in the original design.

**Review by iropolo — APPROVED** [review]

**Comment by fede-dp** [comment]
love to see this examples! wondering if we can show something in the ones that are empty, I think this is a great starting point for this!

**Comment by fede-dp** [comment]
the negatives ones doesn't makes sense to me because we don't have a 'starting point' but i think is technically correct in the example

**Review by fede-dp — APPROVED** [review]

**Pull request merged**